Detectors/CTP/workflow/src/RawToDigitConverterSpec.cxx

```diff
@@ -70,6 +70,7 @@ void RawToDigitConverterSpec::run(framework::ProcessingContext& ctx)
   for (auto it = parser.begin(); it != parser.end(); ++it) {
     auto rdh = it.get_if<o2::header::RAWDataHeader>();
     auto triggerOrbit = o2::raw::RDHUtils::getTriggerOrbit(rdh);
+    std::cout << "==================>" << std::hex << triggerOrbit << std::endl;
```

Comment:
this will create a huge loge, please remove.

Reply:
Sorry, forgot it. Noe removed.

Reply:
It seems you did not push updated code, please do so, if this is the only change, I'll merge it w/o waiting for the new CI